public/app/features/dashboard-scene/serialization/DashboardSceneSerializer.ts

```diff
@@ -400,10 +411,14 @@ export class V2DashboardSerializer
     return this.metadata;
   }
 
-  getTrackingInformation(s: DashboardScene): DashboardTrackingInfo | undefined {
+  getTrackingInformation(s: DashboardScene): (DashboardTrackingInfo & DashboardV2TrackingInfo) | undefined {
     if (!this.initialSaveModel) {
       return undefined;
     }
+    // type guard for initialSaveModel as DashboardV2Spec
+    if (!isDashboardV2Spec(this.initialSaveModel)) {
```

Comment:
I feel like this should be fixed a little bit. `getTrackingInformation` is used also for V1 dashboards. So, instead of just returning, we can append the Dashboard V2 tracking if we have a V2 dashboard. Otherwise, just return the usual tracking info.